cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1584,9 +1584,9 @@ function Install_Containerd {
   $tmp_dir = 'C:\containerd_tmp'
   New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
 
-  # TODO(ibrahimab) Change this to a gcs bucket with CI maintained and accessible by community.
-  $version = '1.4.4'
-  $tar_url = ("https://github.com/containerd/containerd/releases/download/v${version}/" +
+  # TODO(ibrahimab) Change this to pick from a CI maintained and accessible by community.
+  $version = '1.5.2'
+  $tar_url = ("https://storage.googleapis.com/gke-release/winnode/containerd/${version}/" +
```

Comment:
Is this not actually 1.5.2 then based on previous discussion? because the old public download URL should work if it's not patched, and if it is patched, I think the version or some comment or _something_ should reflect that we are not actually testing unmodified 1.5.2 here.

Reply:
This is a patched one. Let me add a separate comment for that as well!

Reply:
Added!, Thanks for suggesting so Benjamin!